### PR TITLE
code comment enabled for translation

### PR DESCRIPTION
**Pull request by bfontaine**
One of the code snippets features a comment in English. This PR enables its translation. Since I don’t know how to translate it in languages other than French I copied the English text in all translations.

### CODE REVIEW

**Comment by mikemcquaid** [comment]
@DomT4 I guess this is an approach we could take in your other PR?

**Comment by xu-cheng** [comment]
If any help, I can translate it into Chinese.

File: index_zh-cn.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
   createpackages: 轻松创建您的 Homebrew 程式。
   hack: 以 git、 ruby 为其筋骨，所以借助您的相关知识，自由修改，并且可以简单撤回您的调改或者合并上游更新。
   formula: Homebrew 的程式都是简单的 Ruby 脚本：
+  editor: opens in $EDITOR!
```
Comment (xu-cheng) [inline index_zh-cn.html]: editor: 使用 $EDITOR 编辑!

File: index_zh-tw.html [hunk 2]
```diff
@@ -10,6 +10,7 @@
   createpackages: 輕鬆建立你自己的 Homebrew packages。
   hack: 完全以 Git 和 Ruby 為基底，所以你可以盡情地運用這些知識，輕鬆地復原你的修改以及合併上游的更新。
   formula: "Homebrew 的 formula 都是簡單的 Ruby 腳本："
+  editor: opens in $EDITOR!
```
Comment (xu-cheng) [inline index_zh-tw.html]: editor: 使用 $EDITOR 編輯!

File: index_zh-cn.html (hunk 1 above)
Comment (bfontaine) [inline index_zh-cn.html]: Thanks, included.

File: index_zh-tw.html (hunk 2 above)
Comment (bfontaine) [inline index_zh-tw.html]: Thanks, included.

File: index_zh-cn.html [hunk 3]
```diff
@@ -10,6 +10,7 @@
   createpackages: 轻松创建您的 Homebrew 程式。
   hack: 以 git、 ruby 为其筋骨，所以借助您的相关知识，自由修改，并且可以简单撤回您的调改或者合并上游更新。
   formula: Homebrew 的程式都是简单的 Ruby 脚本：
+  editor: 使用 $EDITOR 編輯!
```
Comment (xu-cheng) [inline index_zh-cn.html]: @bfontaine You include the wrong one.  It should be `editor: 使用 $EDITOR 编辑!`. FYI, there are two Chinese writing system: Simplified Chinese and Traditional Chinese.

Comment (bfontaine) [inline index_zh-cn.html]: @xu-cheng Fixed. To be honest, I had to zoom in my editor to understand how these are different.

Comment (xu-cheng) [inline index_zh-cn.html]: :wink:

**Pull request merged**